ports/qlementine/vcpkg.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "qlementine",
+  "version": "1.0.2",
+  "description": "Modern QStyle for desktop Qt6 applications.",
+  "homepage": "https://github.com/oclero/qlementine/",
+  "documentation": "https://oclero.github.io/qlementine/",
+  "license": "MIT",
+  "supports": "windows | linux | osx",
+  "dependencies": [
+    "qtbase",
```

Please do not depend on default features of qtbase.
```suggestion
    {
      "name": "qtbase",
      "default-features": false,
      "features": [
        <widgets? declarative? Please specificy.>
      ]
    },
```